src/CommunityToolkit.Maui/Converters/BaseConverterOneWay.shared.cs

```diff
@@ -9,6 +9,13 @@ namespace CommunityToolkit.Maui.Converters;
 /// <typeparam name="TTo">Type of the output value</typeparam>
 public abstract class BaseConverterOneWay<TFrom, TTo> : BaseConverter<TFrom, TTo>
 {
+	/// <inheritdoc/>
+	public override TFrom DefaultConvertBackReturnValue
+	{
+		get => throw new NotSupportedException($"{nameof(DefaultConvertBackReturnValue)} not used for ${nameof(BaseConverterOneWay<TFrom, TTo>)}");
```

Comment:
you know English better than I do. should it be **is** not used?

Reply:
Yup! Fixed ✅

Your English is clearly better than mine 😆🤦‍♂️